lib/lf/geometry/geometry.h

```diff
@@ -2,16 +2,65 @@
 #define __02a3dfa9ae3a4969b29d4c0ecfaa6ad9
 
 /**
+ * @file geometry.h
```

Comment:
This comment was a __namespace comment__. Please don't make it into a file header comment. (FIXED)